tests/test_incremental.py

```diff
@@ -96,22 +99,20 @@ def test_scoring_string(scheduler, xy_classification, scoring):
         assert callable(check_scoring(clf, scoring=scoring))
         clf.fit(X, y, classes=np.unique(y))
         clf.score(X, y)
-        clf.estimator.score(X, y)
```

Comment:
I don't think this was doing what we wanted earlier.

When Incremental is created with `scoring`, `clf.score(X, y)` will in general be different then `clf.estimator_.score(X, y)`. The latter always uses the "default" scoring for that estimator (e.g. accuracy score). I opted to just remove this line.